src/providers/Web3Provider.tsx

```diff
@@ -7,6 +7,7 @@ import { NETWORKS } from '../constants/config'
 import WrappedRoseMetadata from '../contracts/WrappedROSE.json'
 import { TransactionResponse } from '@ethersproject/abstract-provider'
 import { MetaMaskError, UnknownNetworkError } from '../utils/errors'
+// @ts-expect-error TODO: fix it in clean up PR
 import * as detectEthereumProvider from '@metamask/detect-provider'
```

Comment:
This import actually breaks production app - so before merging auto deploy - this need to be fixed. Will create a PR to fix the issue, and the issue bellow.

Reply:
Opened here: https://github.com/oasisprotocol/dapp-wrose/pull/13